src/main/resources/api.yaml

```diff
@@ -207,3 +237,21 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  SprintSummaryDto:
+    type: object
+    properties:
+      participantAvailabilityMapping:
+        type: object
+        additionalProperties:
+          type: integer
+          format: int64
+      totalParticipantsHours:
```

Comment:
skoro userzy podają w minutach, to może tutaj też warto byłoby wysyłać minuty?
poza tym kiedyś już było ustalane, że w nazwie ma nie być jednostki, ewentualnie jako description

Reply:
Racja, zmienię